WhateverGreen/kern_igfx.cpp

```diff
@@ -376,6 +408,44 @@ bool IGFX::processKext(KernelPatcher &patcher, size_t index, mach_vm_address_t a
 			}
 		}
 		
+		if (hdmiP0P1P2Patch) {
+			auto ppp = patcher.solveSymbol(index, "__ZN31AppleIntelFramebufferController17ComputeHdmiP0P1P2EjP21AppleIntelDisplayPathPNS_10CRTCParamsE", address, size);
+			if (ppp) {
+				patcher.eraseCoverageInstPrefix(ppp);
+				orgComputeHdmiP0P1P2 = reinterpret_cast<decltype(orgComputeHdmiP0P1P2)>(patcher.routeFunction(ppp, reinterpret_cast<mach_vm_address_t>(wrapComputeHdmiP0P1P2), true));
```

Comment:
What is the reason for saving the original function when you do not use it? Most likely you want to just route the function without generating a trampoline. Check routeFunction arguments, it has a bool for this. In this case you will also not need `patcher.eraseCoverageInstPrefix`.